packages/atomic/cypress/e2e/breadbox.cypress.ts

```diff
@@ -208,16 +208,20 @@ describe('Breadbox Test Suites', () => {
     });
 
     describe('when selecting "+" show more breadcrumb', () => {
-      before(() => {
+      function setupFacetWithMultipleSelectedValuesAndShowMore() {
         setupFacetWithMultipleSelectedValues();
         BreadboxSelectors.breadcrumbShowMoreButton().click();
-      });
+      }
 
       describe('verify accessibility', () => {
+        beforeEach(setupFacetWithMultipleSelectedValuesAndShowMore);
+
         BreadboxAssertions.assertFocusBreadcrumb('@numberOfVisibleButtons');
       });
 
       describe('verify rendering', () => {
+        before(setupFacetWithMultipleSelectedValuesAndShowMore);
+
```

Comment:
This fix is unrelated to the main issue, but it also failed in this PR. A test failed because it said there was no registered alias for `@numberOfVisibleButtons`.

I couldn't quite figure out why the alias is sometimes registered and sometimes not, but I found [a recommendation from Cypress to use the `beforeEach` hook instead](https://docs.cypress.io/guides/core-concepts/variables-and-aliases#Aliases-are-reset-before-each-test), so I'm trying that.